carp.common/src/commonTest/kotlin/dk/cachet/carp/common/application/TestInstances.kt

```diff
@@ -19,6 +19,7 @@ import kotlin.time.Duration.Companion.milliseconds
  */
 val commonInstances = listOf(
     // `data` namespace.
+    Acceleration( 42.0, 42.0, 42.0 ),
```

Comment:
I still think it should be considered using constrained randomized values rather than the same constants forever. The unit should work with all values within a given range.

Reply:
Here, that would be a horrible idea, since I think these values are even used to verify whether object creation is fully deterministic (always the same values).

For more fuzz testing I can look into that later. These `commonInstances` don't serve that purpose, though.

Reply:
As long as the values used to create the initial object are known to the assertions, I don't see the problem